src/app/pages/LiveRadioPage/index.jsx

```diff
@@ -95,10 +95,10 @@ const LiveRadioPage = ({ pageData }) => {
     translations,
   );
 
-  const { enabled } = useToggle('liveRadioSchedule');
-  // onLiveRadioPage is enabled on Persian to render the schedule for bbc_dari_radio
-  // however bbc_persian_radio should not show the schedule
-  const showSchedule = enabled && radioScheduleOnPage;
+  const { enabled, value } = useToggle('liveRadioSchedule');
+
+  const showSchedule =
+    enabled && RegExp(value).test(service) && radioScheduleOnPage;
```

Comment:
All awesome - as we were messaging about, maybe it would be good to have unit tests for this condition?